src/dataset-builder/ConceptSelector.ts

```diff
@@ -65,13 +67,14 @@ export const ConceptSelector = (props: ConceptSelectorProps) => {
         ],
         initialRows,
         getChildren,
+        datasetId,
       }),
     ]),
     cart.length !== 0 &&
       h(ActionBar, {
         prompt: cart.length === 1 ? '1 concept selected' : `${cart.length} concepts selected`,
         actionText,
-        onClick: () => _.flow(_.map(getConceptForId), onCommit)(cart),
+        onClick: () => _.flow(_.map(getMockConceptForId), onCommit)(cart),
```

Comment:
The cart is still mocking the a get concept by id call. I believe we will want to replace this with a call to a real Concept endpoint getById. Is that right? I added to mock to the name to remind us this is not actually getting the concept by id, but I can add a to do and make a new ticket as well if it doesn't already exist.